mypy/util.py

```diff
@@ -134,3 +140,36 @@ def id(self, o: object) -> int:
             self.id_map[o] = self.next_id
             self.next_id += 1
         return self.id_map[o]
+
+
+if sys.version_info >= (3, 6):
+    PathType = Union[AnyStr, os.PathLike]
+else:
+    PathType = AnyStr
+
+
+def _replace(src: PathType, dest: PathType, timeout: float = 10) -> None:
```

Comment:
Because you don't parameterize the PathType types, they'll be using Any instead of AnyStr.